app/Scripts/RockTurtle.gd

```diff
@@ -56,7 +56,7 @@ func add_damage(value: int) -> void:
 func _on_AttackTimer_timeout() -> void:
 	print("attack timer timeout")
 	attackAnimationPlayer.play("RockTurtleAttack")
-	$TurtleAttackSFX.play()
```

Comment:
why are we removing the sfx?

Reply:
cuz I added it in the animation, felt like it makes more sense now